### PR TITLE
:sparkles: created team member section component#153

### LINKED ISSUE #153: 🎨 Create Team Member Section Carousel
## User stories

As a user, I want a team member section on my about page. 

---

### Summary

As a user, I want a team member section I can use to render/showcase my current team, I want a good experience on the web and phone.

### Description

Design:

Team-member carousel

[Link to full design](https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528)

### Technical Analysis

The team members' section component should be placed in the About page module under the components folder. The component name is team-members-section. The component reuses the carousel component created in #16. The carousel has been updated to allow transclusion so make sure you sync the latest changes. It also reuses the team member component created in #125. 

The component accepts an Angular input of type (already in the codebase):

```typeScript
 TeamMember []
```

### Data

Mock team members are defined for the component and placed in data/sections (already in the codebase) under the file name team-members-list.data.ts. The index file exports a TeamMember[] with the name "__highlightedTeamMembers".

### Acceptance Criteria

- [ ] When rendered the component is pixel-perfect compared to the design above.
- [ ] The component SCSS uses CSS variables for colour, font, ... Connected to the main style.scss
- [ ] The code is isolated to the `libs/elements/layout/cards` package unless a justified reason is specified.
- [ ] The design is responsive for screen, tablet and phone
- [ ] A separate commit is used for each small change
- [ ] The commits and PR are well formatted/use proper documentation
- [ ] The module exports the component

### PR DESCRIPTION
**Pull request by levy-web**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes #153

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## Screenshot (optional)

Screenshot from 2023-09-01 10-11-36

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/pages/about/src/lib/components/team-members-section/team-members-section.component.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import { __highlightedTeamMembers } from '@elewa-website/data/sections';
+
+@Component({
+  selector: 'elewa-website-team-members-section',
+  templateUrl: './team-members-section.component.html',
+  styleUrls: ['./team-members-section.component.scss'],
+})
+export class TeamMembersSectionComponent {
+  highlightedTeamMembers = __highlightedTeamMembers;
```
Comment (Beulah-Matt) [inline libs/features/pages/about/src/lib/components/team-members-section/team-members-section.component.ts:11]: Hi @levy-web 
This needs to be an input, like so: 
@Input() TeamMember: []
This way we can pass the array of teamMembers objects as inputs from the parent.

File: libs/features/pages/about/src/lib/components/team-members-section/team-members-section.component.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class="team-section">
+    <div>
+    <p class="heading">These people make it happen</p>
```
Comment (Beulah-Matt) [inline libs/features/pages/about/src/lib/components/team-members-section/team-members-section.component.html:3]: Add this to translations, like the header and footer content has been done.

**Review by Beulah-Matt — CHANGES_REQUESTED** [review]
Good job. Just make those changes. 
Also, your buttons are too short and close together. See if you can fix that too.

**Comment by levy-web** [comment]
@Beulah-Matt completed the fix kindly check

**Review by Beulah-Matt — APPROVED** [review]
Good work. Approving this.

**Review by LemmyMwaura — APPROVED** [review]

**Pull request merged**